server/management/commands/import_players.py

```diff
@@ -70,13 +70,15 @@ def handle(self, *args: Any, **options: Any) -> None:
                 player_data = {
                     "user": user,
                     "date_of_birth": dob.strftime("%Y-%m-%d"),
-                    "gender": GENDERS.get(row["gender"], Player.GenderTypes.OTHER),
-                    "other_gender": row["other_gender"] if row["other_gender"] != "-" else None,
-                    "city": row["city"],
-                    "state_ut": STATE_UT.get(row["state_ut"], None),
-                    "not_in_india": row["not_in_india"].upper() == "Y",
-                    "occupation": OCCUPATIONS.get(row["occupation"], None),
-                    "educational_institution": row["educational_institution"],
+                    "gender": GENDERS.get(row["gender"].strip(), Player.GenderTypes.OTHER),
```

Comment:
This would become null when the data from the sheet isn't stripped of whitespace